components/Graph.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
+
+const Graph: FC = () => {
+  return (
+    <>
+      <div className="flex-1 my-2 p-4 text-center bg-ocf-gray-900 rounded-2xl p-2.5 dark:bg-ocf-gray-800">
+        <div className="mb-1 text-2xl text-ocf-yellow font-bold">Graph</div>
+        <div className="h-40"></div>
+      </div>
+    </>
```

Comment:
NO Need to fix this (since this component is non-permanent), don't in fact. Just here for reference

Since the root element of this component is a fragment, and the fragment only has one child, the fragment can be removed:

```suggestion
      <div className="flex-1 my-2 p-4 text-center bg-ocf-gray-900 rounded-2xl p-2.5 dark:bg-ocf-gray-800">
        <div className="mb-1 text-2xl text-ocf-yellow font-bold">Graph</div>
        <div className="h-40"></div>
      </div>
```